virtme_ng/run.py

```diff
@@ -1084,6 +1093,25 @@ def _get_virtme_numa(self, args):
         else:
             self.virtme_param["numa"] = ""
 
+    def _get_virtme_numa_distance(self, args):
+        if args.numa_distance is not None:
+            if not args.numa:
+                arg_fail("error: --numa-distance can be used only with --numa", show_usage=False)
```

Comment:
Because the issue is due to a wrong using, maybe `show_usable=False` is not needed?
But up to you, it is also easy to run `vng -h` :)

(Same below for the parsing issue).

Reply:
Yeah, initially I was showing the help, but I thought it was a bit too verbose and it could hide the actual error, so I added `show_usage=False`. :)